tests/conftest.py

```diff
@@ -14,6 +14,8 @@ def pytest_configure(config):
     )
 
     contrib_apps = [
+        'rest_auth',
+        'rest_auth.registration',
```

Comment:
Because this PR is adding rest-auth, not allauth.  It is testing only one scenario of configuration for rest-auth.  allauth is much bigger, and I will need an extension for it also, but one step at a time.

I have of course run local Django project with each major scenario, and gone through the code of these apps to check that the drf-spectacular extension covers all of the branching that is required.

This means there is code which handles scenarios which are not tested in CI.  That seems on par with the other auth extensions.  Having multiple tox testenvs for things in 'contrib' seems overboard.  If you want to each 100% test coverage, there are much lower hanging fruit outside of contrib, which should be addressed before applying that criteria to contrib extensions.

Reply:
i'm **not** talking about coverage or not having tests for `allauth`. that is ok for now. merely saying that it is an optional package and thus should only be included, if the module is available. that is the purpose of `contrib_apps`.  you would potentially break all tests for ``--skip-missing-contrib`` if the package is not there.